src/simulations/paramscan.jl

```diff
@@ -62,59 +68,85 @@ parameters = Dict(
 
 adf, _ = paramscan(parameters, initialize; adata, agent_step!, n = 3)
 ```
+
+To enable `parallel = true`, the dependencies should be loaded as:
+```
+using Distributed
+addprocs(4)
+@everywhere begin
+    using Agents
+    using Random
+    using Statistics: mean
+    using DataFrames
+end
+```
+and also add `@everywhere` to every function or variables needed by the simulations
+```
+@everywhere mutable struct SchellingAgent ...
+@everywhere function agent_step! ...
+@everywhere function initialize ...
+@everywhere happyperc(moods) ...
+@everywhere adata ...
+@everywhere parameters
+```
+then do the parameter scan by
+```
+adf, _ = paramscan(parameters, initialize; parallel = true, adata, agent_step!, n = 3)
+```
+
+To avoid `@everywhere` in everywhere, you can write the core part of the model
+in `Schelling.jl` and do `@everywhere include("Schelling.jl"`
```

Comment:
I would prefer avoiding duplicate information. This already exists in the `[Performance Tips](@ref)` section and thus should not make the docstring here bigger. Please cross-ref to this section in the docstring.

Reply:
Notice that the last two lines you've added are genuinely true information, and you should simply port them to the Performance Tips doc page.

Reply:
Done. 

Maybe a full working example about parallel run should be included in the documentation?